docs/kratos/sdk/go.md

```diff
@@ -0,0 +1,126 @@
+---
+id: go
+title: Go
+---
+
+To install the Go SDK, run:
+
+```
+go get github.com/ory/client-go
+```
+
+## Configuration
+
+We use code generation to generate our SDKs. The Go SDK is generated using
+[`go-swagger`](http://goswagger.io). Set up the SDK:
+
+```go
+import "github.com/ory/client-go/client"
+```
+
+To use a proxy, set the environment variable HTTP_PROXY:
+
+```go
+os.Setenv("HTTP_PROXY", "http://proxy_name:proxy_port")
+```
+
+## Make requests
+
+Making requests is straight forward:
+
+```go
+import (
+  "github.com/ory/client-go/client"
+  "github.com/ory/client-go/client/admin"
+  "github.com/ory/client-go/models"
+)
+
+func main() {
+    KratosPublicEndpoint := url.Parse("https://kratos.localhost:4433")
+
+    flow, _, err := s.KratosPublicEndpoint.V0alpha2Api.GetSelfServiceLoginFlow(r.Context()).Id(flowID).Cookie(cookie).Execute()
+    if err != nil {
+        writeError(w, http.StatusUnauthorized, err)
+        return
+    }
+}
+```
```

Comment:
where does `s` come from? also why declare `KratosPublicEndpoint` when it isn't used?